FV3/atmos_cubed_sphere/tools/fv_diagnostics.F90

```diff
@@ -676,6 +676,12 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
                'theta_e', 'K', missing_value=missing_value )
           idiag%id_omga = register_diag_field ( trim(field), 'omega', axes(1:3), Time,      &
                'omega', 'Pa/s', missing_value=missing_value )
+          idiag%id_vulcan_omga = register_diag_field ( trim(field), 'vulcan_omega', axes(1:3), Time,      &
+               'vulcan_omega', 'Pa/s', missing_value=missing_value )
```

Comment:
Can you add the [CF-recommended](https://cfconventions.org/Data/cf-standard-names/76/build/cf-standard-name-table.html) long name "lagrangian_tendency_of_air_pressure"? OTOH for the non-hydrostatic model, it's actually the lagrangian derivative of the hydrostatic pressure rather than the full pressure.

Reply:
I gave it the long name "Lagrangian tendency of hydrostatic pressure."